Staking to state machine.md

```diff
@@ -1 +1,49 @@
-TODO...
+Staking is a way of earning mining revenue without having to setup a mining machine. Users can buy miner staking tokens to earn a percentage of a mining machine's rewards, either as TEA revenue or TApp token dividend rewards.
+
+The earlier a user buys a staking token, the higher percentage of rewards that's earned. That's because the earlier one buys into a bonding curve, the more chances they have for both price appreciation and a longer duration of dividend rewards from holding the token.
+
+There are two options for users who want to stake to TEA Project mining nodes:
+
+- Staking to B CML nodes is where users earn part of an **individual B CML miner's** revenue stream.
+- Staking to A CML nodes (state machine validators) is where users can earn a share of **all A CML mining nodes'** revenue stream.
+
+Both of these options are exercised through buying the associated mining stake token that's issued along a bonding curve.
+
+- For B CML, a user would buy an individual miner's staking token.
+- For A CML, all users buy the same mining staking token that represents all A CML state machine mining validators.
+
+## Calculating state machine validator rewards
+Unlinke B CML mining nodes who also earn TApp tokens from TApp hosting, the state machine validator nodes earn exclusively in TEA tokens. The following sections will provide more details on the block reward schedule for both state machine validator nodes as well as the B CML nodes. As you can see below, A CML state machine validators earn over 3x as much TEA rewards compared to B CML nodes.
```

Comment:
I think you only compare the public service reward for hosting CMLs. Hosting CMLs main income is from the hosting service. so you might need make it clear